### PR TITLE
Add textarea styles

### PR DESCRIPTION
**Pull request by dpiatek**
Jira Ticket Link / Motivation
----

Migrate textarea styles from website.

Summary of changes
----

See commits for details.

How do you manually test this?
----

As review apps are not currently working for AblyUI, please checkout locally. Textarea has been added in the "Forms section" on the index.

Reviewer Tasks (optional)
----

Merge/Deploy Checklist
----

- [ ] Written automated tests for implemented features/fixed bugs
- [ ] Rebased and squashed commits
- [ ] Commits have clear descriptions of their changes
- [ ] Checked for any performance regressions

Frontend Checklist
----

- [ ] No frontend changes in this PR
- [ ] Added before/after screenshots for changes
- [ ] Tested on different platforms/browsers with [Browserstack](https://www.browserstack.com)
- [ ] Compared with the initial design / our [brand guidelines](https://www.figma.com/file/jTQgyIovrhGBOf4Zrvjvbk/Ably-Linear-Design?node-id=118%3A0)
- [ ] Checked the code for accessibility issues ([VoiceOver User Guide](https://support.apple.com/en-gb/guide/voiceover/welcome/mac))?

### CODE REVIEW

File: preview/app/views/pages/index.html.erb [hunk 1]
```diff
@@ -155,7 +155,7 @@
           <h2 class="text-h1 font-medium mb-40">Typography</h2>
 
           <p class="ui-text-p1 mb-32">Type definitions do not include margin as it can be contextual. Figma and the below list include recommended margins.</p>
-          <p class="ui-text-p1 mb-32">The below list is created with ui-text-* helpers which include properties beyond the the standard <code class="pi-inline-code">font-size</code> and <code class="pi-inline-code">line-height</code> that text-* includes.</p>
+          <p class="ui-text-p1 mb-32">The below list is created with ui-text-* helpers which include properties beyond the the standard <code class="ui-inline-code">font-size</code> and <code class="ui-inline-code">line-height</code> that text-* includes.</p>
```
Comment (artismarti) [inline preview/app/views/pages/index.html.erb:158]: Is there a reason this class is not replaced in https://github.com/ably/ably-ui/blob/cf8fed962bd3f2341fdf3f6493c21e77793f10cd/preview/app/views/components/icon/_arguments.html.erb

**Review by artismarti — CHANGES_REQUESTED** [review]

File: preview/app/views/pages/index.html.erb (hunk 1 above)
Comment (dpiatek) [reply preview/app/views/pages/index.html.erb:158]: nice catch, will update that too 👍🏻

Comment (artismarti) [reply preview/app/views/pages/index.html.erb:158]: Cool! Then that entire css class can be deleted. :)

**Review by artismarti — APPROVED** [review]

File: preview/app/views/components/icon/_arguments.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="max-w-screen-md font-sans font-light">
-  <p class="text-p2 text-cool-black mb-32">Display icons. Uses <code class="pi-inline-code">&lt;use&gt;</code> to insert SVG's. Assumes the spritesheet is already present on the page (this can be done with <code class="pi-inline-code">core/load-sprites.js</code>). Takes the following arguments:</p>
+  <p class="text-p2 text-cool-black mb-32">Display icons. Uses <code class="ui-inline-code">&lt;use&gt;</code> to insert SVG's. Assumes the spritesheet is already present on the page (this can be done with <code class="ui-inline-code">core/load-sprites.js</code>). Takes the following arguments:</p>
```
Comment (artismarti) [inline preview/app/views/components/icon/_arguments.html.erb:2]: the new css class is `ui-textarea`, but the class being used here is `ui-inline-code`

Am I missing some context?

Comment (dpiatek) [reply preview/app/views/components/icon/_arguments.html.erb:2]: Nope, just fixed that along adding the class. I'll make that more obvious by splitting the commits

**Pull request merged**